pyracf/scripts/install_precheck.py

```diff
@@ -0,0 +1,34 @@
+from pyracf import ResourceAdmin, SecurityResponseError
+
+
+def define_precheck_profile():
```

Comment:
Can we rename this script to `setup_precheck.py` and change the name of the function to `setup_precheck()`?